content/blog/why_kubeflow.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Why Kubeflow in your Infrastructure?"
+description = "Answering the 'why' about Kubeflow in an organization's Infrastructure."
+weight = 20
+publishDate = 2018-11-09T02:00:00+01:00
+draft = false
++++
+
+##### Guest post by Ankit Bahuguna & Faheem Nadeem, Software Engineer(s), Cliqz GmbH
```

Comment:
Generally, Kubeflow blog posts present author attribution at the end of the post.